packages/itwinui-css/src/button/base.scss

```diff
@@ -3,36 +3,25 @@
 @use '../mixins';
 
 @mixin iui-button {
-  @include mixins.iui-reset;
-  font-family: inherit;
   display: inline-flex;
   flex-shrink: 0;
   align-items: center;
   vertical-align: middle;
   justify-content: center;
   position: relative;
-  border-radius: var(--iui-border-radius-1);
   line-height: 1.2;
-  font-size: var(--iui-font-size-1);
-  font-weight: var(--iui-font-weight-normal);
   user-select: none;
   cursor: pointer;
   white-space: nowrap;
-  border: 1px solid var(--_iui-button-border-color);
-  background: var(--_iui-button-background-color);
-  color: var(--_iui-button-text-color);
   gap: var(--_iui-button-gap);
-  min-block-size: var(--_iui-button-min-height);
-  min-inline-size: var(--_iui-button-min-height);
   padding-block: var(--_iui-button-padding-block);
   padding-inline: var(--_iui-button-padding-inline);
-  transition:
-    background-color var(--iui-duration-1) ease-out,
-    border-color var(--iui-duration-1) ease-out;
-  -webkit-tap-highlight-color: transparent;
   text-decoration: none;
   @include iui-button-size;
-  @include mixins.iui-focus;
+
+  &:where([disabled], :disabled, [aria-disabled='true'], [data-iui-disabled='true'], .iui-disabled) {
+    cursor: not-allowed;
```

Comment:
I think this is already handled in `iui-field`?

Reply:
Ahh right, I forgot about this.

This line overrides the cursor setting in `field.scss`:
https://github.com/iTwin/iTwinUI/blob/9ab9e8fbbae492bbbc621ee8f9848b5b47f55b5b/packages/itwinui-css/src/button/base.scss#L14